core/io/image_loader.h

```diff
@@ -65,8 +65,6 @@ class ImageLoader {
 	static void add_image_format_loader(ImageFormatLoader *p_loader);
 	static void remove_image_format_loader(ImageFormatLoader *p_loader);
 
-	static const Vector<ImageFormatLoader *> &get_image_format_loaders();
-
```

Comment:
I recall using this for something at some point, but not sure... There would be no way to access image loaders directly anymore.